gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1329,25 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+uintptr_t
+MM_ParallelGlobalGC::clearHeap(MM_EnvironmentBase *env,UDATA walkFlags, uintptr_t walkReason, MM_HeapWalkerObjectFunc walkFunction)
+{
+	memorySizes counter;
+	counter.freeBytes = 0;
+	counter.objectBytes = 0;
+	uintptr_t fixedObjectCount = 0;
+	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
+	omrtty_printf("ClearHeap CALLED.\n");
+	_heapWalker->allObjectsDo(env, walkFunction, &counter, walkFlags, false, false, true);
+	U_64 startTime = omrtime_hires_clock();
+	_extensions->globalGCStats.fixHeapForWalkTime = omrtime_hires_delta(startTime, omrtime_hires_clock(), OMRPORT_TIME_DELTA_IN_MICROSECONDS);
+	_extensions->globalGCStats.fixHeapForWalkReason = walkReason;
+	omrtty_printf("Free Memory size: %lu\n", counter.freeBytes);
+	omrtty_printf("Object size: %lu\n", counter.objectBytes);
```

Comment:
instead of these 2 prints, introduce a single trace point and print them both there